src/test/java/org/broadinstitute/consent/http/resources/ConsentVoteResourceTest.java

```diff
@@ -30,31 +32,125 @@
     private EmailNotifierService emailNotifierService;
 
     private ConsentVoteResource resource;
+    private Vote vote;
+    private Consent consent;
 
     @Before
     public void setUp() {
-        MockitoAnnotations.initMocks(this);
+        openMocks(this);
+
+        consent = new Consent();
+        consent.setConsentId(UUID.randomUUID().toString());
+
+        vote = new Vote();
+        vote.setVoteId(RandomUtils.nextInt(100, 1000));
+        vote.setVote(false);
+        vote.setRationale("Test");
```

Comment:
One thing that I find helpful is to have a generator function instead of using global class variables. What happens over time is that the global var needs to be modified for a new test, which can then unintentionally affect other tests. Instead, if we create a private generator function that works for the current cases, then a new test that needs new info on the same object can call that, modify what it needs, and not break any other tests in the process.